sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -176,7 +176,7 @@ def _next_message_in_buffer(self):
         # pylint:disable=protected-access
         message = self._message_buffer.pop()
         event_data = EventData._from_message(message)
-        trace_link_message(event_data)
+        get_event_links(event_data)
```

Comment:
I'm wondering if this line has any effect..`get_event_links` would return links but we don't use?

Reply:
just verified - removed this line - there's no effect of this